src/Tizen.NUI.Scene3D/src/public/Light/Light.cs

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Runtime.InteropServices;
+using System.ComponentModel;
+using Tizen.NUI;
+using Tizen.NUI.Binding;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Scene3D
+{
+    /// <summary>
+    /// This class is to define 3D Light source.
+    /// Currently this Light class supports Directional Light that lits every position from the same direction. (e.g, Sun light)
```

Comment:
Minor :
If the verb, `light` is present,
```suggestion
    /// Currently this Light class supports Directional Light that lights every position from the same direction. (e.g, Sun light)
```
Or if the `light` is past,
```suggestion
    /// Currently this Light class supports Directional Light that lit every position from the same direction. (e.g, Sun light)
```

Reply:
You have hawk eye.
Because of my short english skill, how do you think which is clear for the users?

Reply:
Hawk eye kkk :smiley_cat: 
I think all descriptions of this class are written down as present tense, so `lights` is more suitable for me :)

Reply:
Thank you English expert.